addons/mail/static/src/components/attachment/attachment.xml

```diff
@@ -10,18 +10,20 @@
                 'o-temporary': attachment and attachment.isTemporary,
                 'o-viewable': attachment and attachment.isViewable,
             }" t-att-title="attachment ? attachment.displayName : undefined" t-att-data-attachment-local-id="attachment ? attachment.localId : undefined"
+                role="menu" t-att-aria-label="attachment and attachment.displayName"
```

Comment:
guideline:
attribute order: keep `data-` at the end (comment applies to whole diff)
and don't create a new line.

Reply:
Done